examples/funloc/analysis_fun.py

```diff
@@ -95,24 +102,25 @@
 # Set what will run
 mnefun.do_processing(
     params,
-    fetch_raw=True,  # Fetch raw recording files from acq machine
-    do_score=True,  # do scoring
+    fetch_raw=True,     # Fetch raw recording files from acquisition machine
+    do_score=True,      # Do scoring to slice data into trials
     # Make SUBJ/raw_fif/SUBJ_prebad.txt file with space-separated
     # list of bad MEG channel numbers, needed for running SSS.
```

Comment:
@Eric89GXL, do these (and some of the other full line comments below) belong? I'm not exactly sure what they're supposed to convey.